nodejs/week1/src/backend/app.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+
+/* 
+/meals
+Respond with the json for all the meals. For each meal, if there are reviews matching it's meal ID, 
+add these reviews to each meal in the form of an array. For meals that do not have any reviews,
+the "reviews" property will be an empty array. (watch the GIF below to understand how it should be 
+structured)
+*/
+
+const copiedMeals = meals;
+const mealsWithReviews = copiedMeals.map((meal) => {
```

Comment:
you can map on meals as map returns a new array.

Reply:
HI Aina
I have fixed your comments in my code